'DVC-file(s)' replaced with '.dvc file(s)' (#1372)
iterative/dvc.org: content/docs/command-reference/init.md

- Not enough isolation/granularity - commands like `dvc pull`, `dvc checkout`,
  and others analyze the whole repository to look for

>   [`.dvc` files](/doc/user-guide/dvc-file-format) to download files and

Reviewer: ```suggestion
  [`.dvc` files or `dvc.yaml`](/doc/user-guide/dvc-file-format) to download files and
```
> p needs wrapping.

- Not enough isolation/granularity - commands like `dvc pull`, `dvc checkout`,
  and others analyze the whole repository to look for

>   [`.dvc` files](/doc/user-guide/dvc-file-format) or
>   [`dvc.yaml`](/doc/user-guide/dvc-file-format) to download files and

Reviewer: ```suggestion
  [`dvc.yaml`](/doc/user-guide/dvc-file-format) or
  [`.dvc`](/doc/user-guide/dvc-file-format) files to download files and
```

might happen that Git repository root is located at different path than the DVC
project root.)

DVC project root defines the scope for most DVC commands. Mostly meaning that

> `dvc.yaml` and all `.dvc` files under the root path are being analyzed.

Reviewer: ```suggestion
all `dvc.yaml` and `.dvc` files under the root path are being analyzed.
```

project root.)

> DVC project root defines the scope for the most DVC commands. Mostly meaning

Reviewer: ```suggestion
DVC project root defines the scope for most DVC commands. Mostly meaning
```

> that all `.dvc` file under the root path are being analyzed.

Reviewer: ```suggestion
that all `.dvc` files and `dvc.yaml` under the root path are being analyzed.
```

Reviewer: @shcheklein (correct Ivan)/ @efiop do you know if `dvc.yaml` can be in a subdirectory? Or additional `dvc.yaml` files in subdirs will be supported soon?

Reviewer 2: @jorgeorpinel Yes, dvc.yaml's can be in any directory inside of the project. You can have multiple of them. Already supported.

Reviewer: Ah! Thanks for the info Ruslan 🙂

Reviewer: Moved this one to https://github.com/iterative/dvc.org/pull/1372#discussion_r435676484

- SCM other than Git is being used. Even though there are DVC features that
  require DVC to be run in the Git repo, DVC can work well with other version
  control systems. Since DVC relies on simple text

>   [`.dvc` files](/doc/user-guide/dvc-file-format) to manage

Reviewer: It's "simple `dvc.yaml` files" here. But linked, please.

Reviewer: Hm. Wasn't totally fixed. Moved to https://github.com/iterative/dvc.org/pull/1372#discussion_r435676718 and https://github.com/iterative/dvc.org/pull/1372, which I'll commit.

- SCM other than Git is being used. Even though there are DVC features that
  require DVC to be run in the Git repo, DVC can work well with other version

>   control systems. Since DVC relies on simple text

Reviewer: ```suggestion
  control systems. Since DVC relies on simple
```

>   [`dvc.yaml`](/doc/user-guide/dvc-file-format) to manage

Reviewer: ```suggestion
  [`dvc.yaml`](/doc/user-guide/dvc-file-format) files to manage
```
